examples/uniaxialCompression/uniaxialCompression.m

```diff
@@ -19,31 +19,40 @@
 %md\textbf{E} = \frac{1}{2}(\textbf{C} - \textbf{I}) = \left[  \begin{matrix} \frac{1}{2} \left(\alpha^2 -1\right) & 0 & 0 \\ 0 &  \frac{1}{2} \left(\beta^2 -1\right) & 0 \\ 0 & 0 &  \frac{1}{2} \left(\beta^2 -1\right) \end{matrix}
 %md\right]
 %md```
+%md where $\alpha = (1+u_x/L_x)$ and $\beta = (1+u_y/L_y)$, with $u_x$ and $u_y$ the linear displacements at $X=(L_x,L_y,L_z=L_y)$.
+%md
 %mdThe neo-Hookean elastic strain energy potential $\Psi$ is given by:
 %md```math
-%md\Psi(\mathbf{C=\textbf{F}^T\textbf{F}}) = \frac{\mu}{2}(I_1 -3) + K ( J^2 -1 )
+%md\Psi(\mathbf{C=\textbf{F}^T\textbf{F}}) = \frac{\mu}{2}(I_1 -2ln(J)) + \frac{K}{2} ( J -1 )^2
 %md```
 %mdwhere $I_1 = \mathrm{tr}(\mathbf{C})$ is the first invariant, $J = \sqrt{\det(\mathbf{C})}$ and $K$ and $\mu$ are the bulk and shear material parameters, respectively.
 %md
 %mdThe second Piola-Kirchhoff tensor is given by:
 %md```math
 %md\textbf{S}( \mathbf{C} ) = \mu (\textbf{I} - \mathbf{C}^{-1}) + K (J(J-1)\mathbf{C}^{-1}) 
 %md```
-%mdthen, using the relation $\textbf{P}=\textbf{F}\textbf{S}$, the $P_{xx}$ nominal stress component is obtained and equaled to the applied compression:
+%mdthen, on the one hand, using the relation $\textbf{P}=\textbf{F}\textbf{S}$, the $P_{xx}$ nominal stress component is obtained and equaled to the applied compression:
 %md```math
 %mdP_{xx}( \mu,K ) =
 %md \alpha \left( 
 %md    \mu -  \frac{\mu}{\alpha^2} + \frac{K\beta^2}{\alpha} (\beta^2 \alpha -1) 
 %md \right) = - p
 %md```
+%md and on the other hand, the $P_{yy}$ and $P_{zz}$ components are obtained and equaled to zero:
+%md```math
+%mdP_{yy}( \mu,K ) =
+%md \beta \left( 
+%md    \mu -  \frac{\mu}{\beta^2} + K (\alpha^2\beta^2 - \alpha) 
+%md \right) = 0
+%md```
 %md
 %md## Numerical solution
 %mdBefore defining the structs, the workspace is cleaned, the ONSAS directory is added to the path and scalar geometry and material parameters are defined.
 clear all, close all
 % add path
 addpath( genpath( [ pwd '/../../src'] ) ) ;
 % scalar parameters
-E = 1 ; nu = 0.3 ; p = -5 ; Lx = 2 ; Ly = 1 ; Lz = 1 ;
+E = 4 ; nu = 0.3 ; p = -5 ; Lx = 2 ; Ly = 1 ; Lz = 1 ;
```

Comment:
related to https://github.com/ONSAS/ONSAS.m/pull/605#issuecomment-1424130077

Reply:
Done